### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by Hiderd** [windicss/windicss#913] [Q&A]
我在启动了vite使用windicss插件的时候，写了类样式，是需要重启我的vite然后再启动才能将样式给到元素

When I started using the Windicss plugin in vite, I wrote a class style that requires restarting my vite and then restarting it to assign the style to the element

**Comment by idamon** [comment]
+1, 遇到同样的问题，修改样式后，页面不刷新，需要重启vite cli才能正常显示页面修改后的样式，无论怎样清除浏览器缓存都没用
